src/Analyzer/FileVisitor.php

```diff
@@ -228,6 +228,7 @@ public function getClassDescriptions(): array
     public function clearParsedClassDescriptions(): void
     {
         $this->classDescriptions = [];
+        $this->classDescriptionBuilder->clear();
```

Comment:
@AlessandroMinoccheri this one line is basically the fix for this issue that I reported 2 years ago 🥹. In order for the tests to run on pipelines I had to run CS Fixer too. Please review

Reply:
@micheleorselli the Mr now contains only necessary bug changes